video.js

```diff
@@ -0,0 +1,90 @@
+import { onBindFirstLoad } from '../utils/jQueryBindFirst';
```

Comment:
Why are you adding this new file `video.js` instead of modifying the existing file `modules/shared/video.js`? You shouldn't have to add a new file for this. As of now, this file wouldn't even be used by WordPress, unless you changed something else in your code that is not part of this pull request.

Reply:
I did change the existing video.js (in modules/shared). Apperently I did something wrong when uploading it to the git.